### PR TITLE
Sequelize upgrade to v5.0.0

### PR DESCRIPTION
**Pull request by akshayKrSingh**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Affected core subsystem(s)
- **egg-sequelize**: `sequelize` dependency upgraded to version `5.0.0`

##### Description of change
Upgrade `sequelize` dependency for `egg-sequelize` pluggin to version 5.0.0.

### CODE REVIEW

**Comment by sm2017** [comment]
@dead-horse @atian25 @fengmk2  @fengmk2  can you please review the PR?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "egg-sequelize",
-  "version": "4.3.1",
```
Comment (dead-horse) [inline package.json]: please don't modify version here. others LGTM

Comment (sm2017) [reply package.json]: @akshayKrSingh can you please fix it?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  "version": "4.4.1",
```
Comment (atian25) [inline package.json]:
```suggestion
  "version": "4.3.1",
```

**Comment by sm2017** [comment]
@dead-horse sorry for mentioning again
Please accept it, release a version

**Comment by dead-horse** [comment]
5.0.0 released

**Pull request merged**